src/Nethermind/Nethermind.Runner/NethermindPlugins.cs

```diff
@@ -0,0 +1,30 @@
+// SPDX-FileCopyrightText: 2025 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+using System.Collections.Generic;
+
+namespace Nethermind.Runner;
+
+public static class NethermindPlugins
+{
+    public static IReadOnlyList<Type> EmbeddedPlugins =
```

Comment:
Fix CA2211 (https://learn.microsoft.com/en-gb/dotnet/fundamentals/code-analysis/quality-rules/ca2211)

```suggestion
    public static readonly IReadOnlyList<Type> EmbeddedPlugins =
```